dcgpy/_graphviz_visualize.py

```diff
@@ -0,0 +1,132 @@
+def _graphviz_visualize(self, erc = 0, draw_inactive = True):
+    """
+    ex.visualize(self, erc, draw_inactive)
+
+    Visualizes the d-CGP expression
+
+    Visualizes the graph of the d-CGP expression, by generating a png image and displaying it on Matplotlib axes
+
+    Note:
+        This method requires matplotlib and pygraphviz modules installed in your Python system
+
+    Args:
+        erc (an ``int``): number of ephemeral random consants in input
+        draw_inactive (a ``Boolean``): indicates whether to draw inactive nodes
+
+    Returns:
```

Comment:
Also does this not also create a png in the current directory? If so should the file name be a kwarg too? by default put the name you have now.